### DISCUSSION TITLE
科技進軍到化妝市場

### DISCUSSION THREAD

**Discussion by Zoe06060606** [1101-GS4719-C/Python-Community-GS4719-C#1087] [科技新知]
文章中提到現在已經能利用圖像辨識出臉型以及分析出最適合的眉型並列印在臉上。
我看到這篇新聞的時候真的是嚇到，沒想到連化妝都可以一鍵解決，對於手殘黨真的是一大福音，希望以後連保養跟全妝都可以坐在椅子上操作手機然後等待機器完成🤣

https://tw.news.yahoo.com/防打呼-智慧枕頭-問世-完美眉型不用畫用印的-113039202.html

**Comment by Doreen331** [comment]
感覺時尚界最近為了與時俱進開始積極的和科技業跨域合作，例如我之前去聽了今年的L'OREAL Brandstorm商業競賽說明，他們今年的主題就是crack the new codes of beauty，要利用科技應用向他們的公司提出一個proposal，我個人是覺得這兩個領域的結合不是大好就是大壞，畢竟化妝還是一件以人為出發點的事

**Comment by peggy0329** [comment]
蠻好奇一掃過臉就印好的眉毛是用什麼方式印象去的，個人認為有時候太過端正的眉型為讓人覺得怪怪的哈
輕手畫得沒那麼完美感覺比較讓人親近

**Comment by wenchi9023** [comment]
美感這件事確實滿主觀的，像有些人就會批評某些韓國女團的化妝師真的可以下台了，怎麼可以把偶像畫成這樣XD
如果這項技術可以幫助找到每個人適合的眉型、甚至妝容，那我認為不管是偶像團體化妝師或我們平民百姓就能以那個目標前進，精進自己的化妝技巧，找到最適合自己的妝容，以免未來回頭看發現都是黑歷史...

**Comment by Sylvie0601** [comment]
我覺得應該可以發展成幫助大家找到自己適合的妝容ㄟ!像是我自己新手化妝都畫的超級可怕，如果有機器可以幫我畫好我的妝容，然後教我如何畫我再自己畫感覺比較讚!

**Comment by yuxinlin93** [comment]
我還看看過可以幫助手抖寶貝畫口紅的輔助器，有點類似香雞穩定器上裝口紅，不管怎麼轉都不會畫歪，對於手不方便，或是有些疾病導致無法控制動作的人真的是一大福音

**Comment by BlahFish1006** [comment]
看報導中的美國CES消費電子展感覺有好多有趣的新科技應用，有機會蠻想親自去觀摩學習的XD
這讓我回想起以前曾經有過的想法：要是服裝或美妝賣場中，能有一個機器，能夠辨識人臉和身形，
模擬出實際上身或上妝的樣子就好了，感覺可以省下很多排隊試衣服得的時間和試妝的成本，
也能幫助消費者挑選到最適合自己的產品，感覺會很有意思。

**Comment by harrylee1971** [comment]
該不會這是一個取代掉美容師行業的AI，現在是每個行業都岌岌可危了嗎
